website/src/App.js

```diff
@@ -19,6 +19,11 @@ import type {
   NavigationScreenComponent,
 } from 'react-navigation';
 
+export type ScreenOptions = {
```

Comment:
Yeah, however separate scenes itself are not yet flow-typed. Will be useful when we actually do it!